datastore.go

```diff
@@ -0,0 +1,11 @@
+package testutil
+
+import (
+	"github.com/ipfs/go-datastore"
+	syncds "github.com/ipfs/go-datastore/sync"
+	ds2 "github.com/ipfs/go-ipfs/thirdparty/datastore2"
```

Comment:
need to update this import, but is this code actually used anywhere? we might just be able to remove this file